utils/mock-data-loader/user-helper.js

```diff
@@ -8,7 +7,7 @@ const mockDataLoaderUser = {
   surname: 'DataLoader',
   roles: ['editor', 'maker'],
   email: 're-insert-mocks-data-loader@ukexportfinance.gov.uk',
-  bank: MOCK_BANKS.find((bank) => bank.id === '9'),
+  bank: { id: '*' },
```

Comment:
Are we inserting a super user here?

Reply:
yes.  Not an issue on GHA but locally, portal-api tests insert deals from a different bank which throw an authentication error if not a super user (due to bank id mismatch)